jujugui/static/gui/src/app/components/user-profile/model-list/test-model-list.js

```diff
@@ -292,171 +278,6 @@ describe('UserProfileModelList', () => {
-        users={users}
-        user={users.charmstore} />, true);
-    var output = component.getRenderOutput();
-    var instance = component.getMountedInstance();
-    // Set up the component to simulate user action.
-    instance.refs = {
-      modelName: {
-        validate: _ => false
-      }
-    };
-    var preventable = { preventDefault: sinon.stub() };
-    // Call the action method in the proper element.
-    output.props.children[0].props.children[2].props.children.props.children[2]
-      .props.action(preventable);
-    assert.equal(
-      preventable.preventDefault.callCount, 1, 'default not prevented');
-    assert.equal(showConnectingMask.callCount, 0, 'mask shown');
-    // Make sure that it switches to the model after it's created.
-    assert.equal(switchModel.callCount, 0, 'model should not be switched to');
-  });
-
-  it('gracefully handles errors when creating new model', () => {
-    // This test doesn't check the user interactions and animations, that
-    // will need to be done with the uitest suite.
-    controllerAPI.createModel = (modelName, userName, callback) => {
-      assert.equal(modelName, 'newmodelname', 'model name not set properly');
-      assert.equal(userName, 'test-owner', 'user name not set properly');
-      // Simulate the model being created.
-      callback({
-        err: 'this is an error',
-        uuid: 'abc123',
-        name: modelName
-      });
-    };
-    var hideConnectingMask = sinon.stub();
-    var addNotification = sinon.stub();
-    var component = jsTestUtils.shallowRender(
-      <juju.components.UserProfileModelList
-        addNotification={addNotification}
-        canCreateNew={true}
-        controllerAPI={controllerAPI}
-        hideConnectingMask={hideConnectingMask}
-        listModels={sinon.stub().callsArgWith(0, null, {models: models})}
-        showConnectingMask={sinon.stub()}
-        switchModel={sinon.stub()}
-        users={users}
-        user={users.charmstore} />, true);
-    var output = component.getRenderOutput();
-    var instance = component.getMountedInstance();
-    // Set up the component to simulate user action.
-    instance.refs = {
-      modelName: {
-        validate: _ => true,
-        getValue: _ => 'newmodelname'
-      }
-    };
-    var preventable = { preventDefault: sinon.stub() };
-    // Call the action method in the proper element.
-    output.props.children[0].props.children[2].props.children.props.children[2]
-      .props.action(preventable);
-    // Make sure that the mask is hidden and that a notification was added
-    // with the error message.
-    assert.equal(hideConnectingMask.callCount, 1, 'mask not hidden');
-    assert.equal(addNotification.callCount, 1, 'notification not added');
-    assert.deepEqual(addNotification.args[0][0], {
-      title: 'Failed to create new Model',
-      message: 'this is an error',
-      level: 'error'
-    });
-  });
-
-  it('swtches models by switching to disconnected with JIMM', () => {
```

Comment:
This test isn't replicated in the new suite - I'm not entirely sure how this functionality will work right now though with the lack of jem and jimmURL not being passed through though - see above comments